CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 2.17.0 (2020-07-28)
+
+- Add support for showing Footer section in also Landscape mode
```

Comment:
```suggestion
- Add support to show Footer section in Landscape mode. 
```

Reply:
Thanks for correcting me here, this sounds more right 👍